addok/http/base.py

```diff
@@ -82,6 +83,7 @@ def to_geojson(self, req, resp, results, query=None, filters=None,
             "features": [r.format() for r in results],
             "attribution": config.ATTRIBUTION,
             "licence": config.LICENCE,
+            "addok": VERSION,
```

Comment:
> I'm more interested in the version of the format that of the formatter so I would not add it to the payload.

We still could have both, they have different meaning ;)
But okay, dropped the commit!